playwright.config.mjs

```diff
@@ -1,92 +0,0 @@
-// @ts-check
-const { defineConfig, devices } = require('@playwright/test');
```

Comment:
I renamed to `.js` and switched to ES module `import` syntax. This was confusing, I had to copy the TypeScript example, even though we're not using TS. But it somehow makes the normal ES6 module stuff work